integration/integrate-with-cloud-sources.adoc

```diff
@@ -0,0 +1,37 @@
+[id="integrate-with-cloud-sources"]
+= Integrating with cloud sources
+include::modules/common-attributes.adoc[]
+:context: integrate-with-cloud-sources
+
+toc::[]
+
+[role="_abstract"]
+You can integrate {product-title} with different cloud sources to discover potential clusters
```

Comment:
I think you use `{rh-rhacs-first}` for the first time you use the product title in a docs page (at least from the limited things I've read).